docs/source/daml/intro/5_Restrictions.rst

```diff
@@ -66,34 +66,38 @@ For example, the simple Iou in :ref:`simple_iou` allowed the no-op where the ``o
 
-There's also quite a lot going on inside the ``do`` block of the ``Redeem`` choice, with several uses of the ``<-`` operator. ``do`` blocks and ``<-`` deserve a proper explanation at this point.
+There are quite a few new time-related functions from the ``DA.Time`` and ``DA.Date`` libraries here. Their names should be reasonably descriptive so how they work won't be covered here, but given that Daml assumes it is run in a distributed setting, we will still discuss time in Daml.
 
 Time on Daml Ledgers
 --------------------
 
-Each transaction on a Daml ledger has two timestamps called the *ledger time (LT)* and the *record time (RT)*. The ledger time is set by the participant, the record time is set by the ledger.
+Each transaction on a Daml ledger has two timestamps. One of the timestamps is called the *ledger time (LT)*. The other is called the *record time (RT)*.
+
+**Ledger time (LT)** is the time associated with a transaction in the abstract ledger model. It is the time of a transaction from a business and application perspective. The LT time is determined by the participant. When you call ``getTime``, it is the LT time that is returned. The LT is used when reasoning about related transactions and commits. The LTs can be compared with other LTs to guarantee model consistency. For example, LTs are used to enforce that no transaction depends on a contract that does not exist. This is the requirement known as "causal monotonicity."
 
-Each Daml ledger has a policy on the allowed difference between LT and RT called the *skew*. The participant has to take a good guess at what the record time will be. If it's too far off, the transaction will be rejected.
+**Record time (RT)** is the time assigned by the persistence layer. It represents the time that the transaction is “physically” recorded. For example, “The backing database ledger has assigned the timestamp of such-and-such time to this transaction.” For a Daml transaction to be valid, it must have an LT and RT that are not too different from each other. This is the only purpose in the RT - to ensure that transactions are being recorded in a timely manner. The RT should  roughly align with the LT. This is the requirement known as “bounded skew.”
 
-``getTime`` is an action that gets the LT from the ledger. In the above example, that time is taken apart into day of week and hour of day using standard library functions from ``DA.Date`` and ``DA.Time``. The hour of the day is checked to be in the range from 8 to 18.
+Each Daml ledger has a policy on the allowed difference between LT and RT called the *skew*. If it's too far off, the transaction will be rejected. Consequently, the participant keeps a running estimate of the time required to record a transation. The participant assigns the LT based on that estimate.
```

Comment:
I don't believe this is universally true. In fact, my understanding is that the LT actually comes from the application as a part of the command? Or has that changed since I last looked at that aspect.

```suggestion
Each Daml ledger has a policy on the allowed difference between LT and RT called the *skew*. If it's too far off, the transaction will be rejected.
```

Reply:
Fixed with 2e4aa2da89cdde635c9b729ef5f7f13f0adf0a22